client-app/pages/compare-products/compare-products.vue

```diff
@@ -0,0 +1,213 @@
+            <span v-if="isMobile" class="block font-extrabold text-sm">{{ key }}</span>
+            {{ value.value }}
+          </div>
+        </div>
+        <div class="flex items-center space-x-5 px-5 lg:px-0">
+          <div class="w-1/6" v-if="!isMobile"></div>
+          <div v-for="product in products" :key="product.id" class="w-32 flex-shrink-0 lg:flex-shrink md:w-48">
+            <!-- Product price -->
+            <div class="flex flex-col md:flex-row items-baseline text-sm my-4">
+              <div class="w-1/2 font-bold text-xs" v-if="!isMobile">
+                {{ $t("pages.compare.main_block.price_label") }}
+              </div>
+              <div class="md:w-1/2">
+                <span class="text-green-700 font-extrabold"><VcPriceDisplay :value="product.price?.actual" /></span>
+                {{ $t("common.suffixes.per_item") }}
+              </div>
+            </div>
+            <AddToCart :product="product" />
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { VcBreadcrumbs, VcCheckbox, VcButton, IBreadcrumbs, VcImage, VcPriceDisplay } from "@/components";
+import { EmptyComparison, useProducts } from "@/shared/catalog";
+import { AddToCart } from "@/shared/cart";
+import SeoUrl from "@core/seo-routes.enum";
+import _ from "lodash";
+import { onMounted, ref, watch } from "vue";
+import { breakpointsTailwind, useBreakpoints } from "@vueuse/core";
+import { useCompareProducts } from "@/shared/compare";
+import { Product as ProductType } from "@/core/api/graphql/types";
+const { fetchProducts, products } = useProducts();
+const { clearCompareList, productsLimit, removeFromCompareList, productsIds } = useCompareProducts();
+
+const breadcrumbs = ref<IBreadcrumbs[]>([
+  { title: "Home", url: "/" },
+  { title: "Compare products", url: "/compare-products" },
+]);
+
+const breakpoints = useBreakpoints(breakpointsTailwind);
+const isMobile = breakpoints.smaller("lg");
+
+const showOnlyDifferences = ref(false);
+
+const originalProperties = ref<{ [key: string]: { value: string }[] }>({});
+const computedProperties = ref<{ [key: string]: { value: string }[] }>({});
+
+const removeProduct = (product: ProductType) => {
+  removeFromCompareList(product);
+};
+
+const clearList = () => {
+  clearCompareList();
+};
+
+const onShowOnlyDifferencesChange = () => {
+  if (showOnlyDifferences.value) {
+    _.each(_.keys(computedProperties.value), (key) => {
+      const values = _.map(_.values(computedProperties.value[key]), (value) => {
+        return value.value;
+      });
+      const uniqueValues = _.uniq(values);
+
+      if (uniqueValues.length === 1) {
+        delete computedProperties.value[key];
+      }
+    });
+  } else {
+    computedProperties.value = { ...originalProperties.value };
+  }
+};
+
+function getProductProperties() {
+  if (_.isEmpty(products.value)) return;
+  const grouped: { [key: string]: { value: string }[] } = {};
```

Comment:
This is the same type as on the above comment, better to use type or interface instead

Reply:
done